administrator/components/com_associations/views/associations/tmpl/default.php

```diff
@@ -16,6 +16,13 @@
 
 $listOrder = $this->escape($this->state->get('list.ordering'));
 $listDirn  = $this->escape($this->state->get('list.direction'));
+
+$colSpan = $this->menuType == true ? 5 : 4;
+
+if ($this->menuType == '')
+{
+	$colSpan--;
+}
```

Comment:
colspan should be dynamic, 4 when you don't have the menutype column, 5 when you have it